app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less

```diff
@@ -113,7 +113,7 @@
 
         .page-main & {
             .block {
-                margin-bottom: 0;
+                margin-bottom: @indent__base;
```

Comment:
Please check my comment as this breaks other parts of summary box margins.

Reply:
Thank you @krzksz and @niravkrish it is done